src/gui_common/tooltip/ToolTipManager.tscn

```diff
@@ -5847,6 +5847,35 @@ rect_min_size = Vector2( 20, 20 )
 mouse_filter = 2
 expand = true
 
+[node name="Can Engulf" type="HBoxContainer" parent="Holder/membraneSelection/cellulose/MarginContainer/VBoxContainer/ModifierList"]
+margin_top = 72.0
+margin_right = 318.0
+margin_bottom = 92.0
+mouse_filter = 2
+script = ExtResource( 21 )
+
+[node name="Value" type="Label" parent="Holder/membraneSelection/cellulose/MarginContainer/VBoxContainer/ModifierList/Can Engulf"]
+margin_top = 1.0
+margin_right = 27.0
+margin_bottom = 18.0
+custom_fonts/font = SubResource( 7 )
+custom_colors/font_color = Color( 1, 0.301961, 0.301961, 1 )
+text = "Can not engulf"
```

Comment:
Also a suggestion, currently the texts are a bit small. I think they would be more noticeable if they are set on the Name label node instead, since it has a bold font. Here's a comparison:
![comparison](https://user-images.githubusercontent.com/54026083/97104207-969a9600-16e4-11eb-886c-498580b4f57b.png)